scene/main/node.cpp

```diff
@@ -497,6 +499,16 @@ void Node::_propagate_pause_owner(Node *p_owner) {
 	}
 }
 
+void Node::_propagate_pause_change(bool can_process) {
+	notification(can_process ? NOTIFICATION_UNPAUSED : NOTIFICATION_PAUSED);
```

Comment:
You can just send the notification as a parameter, instead of checking this condition for every call.